webpack.config.js

```diff
@@ -0,0 +1,7 @@
+const createExpoWebpackConfigAsync = require('@expo/webpack-config')
+
+module.exports = async function (env, argv) {
+  const config = await createExpoWebpackConfigAsync(env, argv)
+  config.resolve.alias['victory-native'] = 'victory'
```

Comment:
Fix plot when using web version with alias as in https://github.com/FormidableLabs/victory-native/issues/573